tests/drivers/spi/spi_loopback/testcase.yaml

```diff
@@ -87,9 +91,9 @@ tests:
     platform_allow:
       - nucleo_h743zi
       - nucleo_h753zi
-  drivers.spi.stm32_spi_16bits_frames_dma_no_nocache.loopback:
+  drivers.spi.stm32_spi_16bits_frames_dma_dt_nocache_mem.loopback:
```

Comment:
It's not clear to me why the 8bits version of the test is restricted to H7, while the 16bits applies to all family

Reply:
Should be restricted to F7 & H7.